### PR TITLE
fix(ui): more consistent alert colors

### PR DESCRIPTION
**Pull request by Chrissy**
alert-colors
almost-there

### CODE REVIEW

File: src/sentry/static/sentry/app/components/alert.jsx [hunk 1]
```diff
@@ -59,10 +45,13 @@ const AlertWrapper = styled('div', {shouldForwardProp: isPropValid})`
...
-    p.system
-      ? p.type && getSystemAlertColorStyles(p.theme.alert[p.type])
-      : p.type && getAlertColorStyles(p.theme.alert[p.type], p.theme.gray5)};
+  a {
+    color: ${p => p.theme.textColor};
+    border-bottom: 1px dotted ${p => p.theme.textColor};
+  }
+
+  ${p => getAlertColorStyles(p.theme.alert[p.type])} ${p =>
+      p.system && getSystemAlertColorStyles(p.theme.alert[p.type])};
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/alert.jsx]: I don't really know how to get prettier to stop chewing this up 🤷‍♀️

File: src/sentry/static/sentry/app/components/alert.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import {cx} from 'emotion';
 import styled from 'react-emotion';
 import isPropValid from '@emotion/is-prop-valid';
+import Color from 'color';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/alert.jsx]: What is this `color` package? I don't see it in `package.json`

File: src/sentry/static/sentry/app/components/alert.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/alert.jsx]: you can try adding a semicolon after maybe

File: src/sentry/static/sentry/app/components/alert.jsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/alert.jsx]: Also should it be imported as `Color` and not `color`?

File: src/sentry/static/sentry/app/utils/theme.jsx [hunk 3]
```diff
@@ -18,17 +18,20 @@ const theme = {
 
   blue: '#3B6ECC',
   blueLight: '#628BD6',
+  blueLightest: '#F5FAFE',
   blueDark: '#2F58A3',
 
   green: '#57be8c',
   greenLight: '#71D8A6',
+  greenLightest: '#f8fcf7',
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/utils/theme.jsx]: 😬 what comes after?

File: src/sentry/static/sentry/app/components/alert.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/alert.jsx]: sorry—I must have dumped my changes to `package.json`. This also probably explains why it isn't passing builds.

**Review by ckj — APPROVED** [review]
👍 on the visuals

File: src/sentry/static/sentry/app/utils/theme.jsx (hunk 3 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/utils/theme.jsx]: greenlighterestest!

jk I think the pattern is:

* colorDarkest
* colorDark
* color
* colorLight
* colorLightest

and we should cap it there. five tints per color should be enough, and if we need something lighter or darker we can consider changing one.

File: src/sentry/static/sentry/app/components/alert.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/alert.jsx]: yes. they capitalize it in the docs but the [whole thing is just a function](
https://github.com/Qix-/color/blob/master/index.js) so I don't know why.

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
@Chrissy I'm rebasing this w/ master for you to get tests rolling

**Comment by Chrissy** [comment]
@billyvg thank you. Builds have been taking so long lately, I start to forget what is out and what isn't 🤣

**Pull request merged**